packages/app-store/routing-forms/components/FormInputFields.tsx

```diff
@@ -70,6 +70,7 @@ export default function FormInputFields(props: FormInputFieldsProps) {
                     ...response,
                     [field.id]: {
                       label: field.label,
+                      identifier: field?.identifier,
```

Comment:
Adds the field identifier to the routing form response. This way we can query directly for the field rather than matching the field id between the form and the response.